### ISSUE TITLE
Can it be OK to call Connect() function directly, after disconnected without any reset work?

### ISSUE THREAD

**Issue by bruce1125** [topfreegames/libpitaya#91]
Hi,
I'm using the unity sdk, but I don't find any reset API for RECONNECT case.
I'm not sure if it will cause some problems.
thanks.

**Comment by felippeduran** [comment]
Hey @bruce1125, do you think this is related to #92

**Comment by bruce1125** [comment]
> Hey @bruce1125, do you think this is related to #92

yeah,I think that is one of the most common issuses which caused by the connection-state management.
